_episodes/05-merging-data.md

```diff
@@ -1,27 +1,22 @@
 ---
-title: Combining DataFrames with Pandas
+title: Combinando DataFrames con Pandas
 teaching: 20
 exercises: 25
 questions:
-  - "Can I work with data from multiple sources?"
-  - "How can I combine data from different data sets?"
+  - ¿Puedo trabajar con datos de diferentes fuentes?
+  - ¿Como puedo combinar datos de diferentes datasets?
```

Comment:
```suggestion
  - ¿Como puedo combinar datos de diferentes conjuntos de datos?
```

Reply:
We have been using _datasets_ in the other lessons and in the [reference](https://github.com/Carpentries-ES/python-ecology-lesson-es/blob/gh-pages/reference.md) rather than translating it to "conjuntos de datos"/